.travis.yml

```diff
@@ -1,26 +1,30 @@
 sudo: required
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 cache:
   directories:
-    - "$HOME/.ivy2/cache"
-    - "$HOME/.sbt/boot/"
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+
+language: java
+
+jdk:
+  - oraclejdk8
```

Comment:
> why we should switch to open JDK, if locally we all use oracle one?

Its only because for OSX openjdk installation is quite dirty affair. In general applications are  deployed and executed on linux environments or are based on linux based docker images which in turn use openjdk.

> duplication removed

Could not find the change. Please try to use for each comment dedicated commit link. It makes review much easier.